src/target/riscv/riscv-013.c

```diff
@@ -5418,16 +5418,22 @@ static int riscv013_step_or_resume_current_hart(struct target *target,
 		return ERROR_OK;
 	}
 
+	LOG_TARGET_ERROR(target, "unable to %s. dmstatus=0x%08x",
```

Comment:
Detail, if you like:

`Failed` is slightly more descriptive than `unable`. It makes it more clear that we tried and it did not go well.

```suggestion
	LOG_TARGET_ERROR(target, "Failed to %s. dmstatus=0x%08x",
```

Reply:
addressed